src/main/webapp/go.js

```diff
@@ -436,6 +445,44 @@ function checkUserDestinationGuess() { //eslint-disable-line
   });
 }
 
+/**
+ * Retrieves the place ID of the current destination, retrieved
+ * using the Places library.
+ */
+function getPlaceIDOfAnswer() {
```

Comment:
There's a lot of overlap between this function and `generatePlaceId()`. Could you factor out the common part to avoid code duplication?